docs/topics/auth/customizing.txt

```diff
@@ -268,6 +268,14 @@ the check performed). An authentication backend will receive the keyword
 parameters ``obj`` and ``user_obj`` for each object related authorization
 method and can return the object level permission as appropriate.
 
+Note that the default :class:`~django.contrib.auth.backends.ModelBackend`
```

Comment:
This first sentence seems to have the same meaning as "That means that checking for object permissions will always return ``False`` or an empty list (depending on the check performed)"  of the previous paragraph.

Aside: I prefer omitting "Note that" as much as possible.